syntax = "proto3";

message ProjectNode {
    string project_name = 1;

    //list of public keys of users that can make changes to this project
    repeated string public_keys = 2;

    repeated string task_names = 3;
}

message ProjectNodeContainer {
    repeated ProjectNode entries = 1;
}
